articles/frontend-is-not-your-enemy.md

```diff
@@ -0,0 +1,157 @@
+---
+authorName: Janko Sokolović
+authorGithubUsername: jasofalcon
+issue: 81
+title: Frontend Is Not Your Enemy 
+---
+# Frontend Is Not Your Enemy
+
+For many years, developers are discriminating frontend technologies because of various reasons.
+
+*“It is too volatile!”*, some said
+
+*“But I've just learned AngularJS version 1.0 yesterday, today it's already v3.3.1”*, many yelled while hiding the tears...
+
+*“It doesn't work the same in every browser!”*, was heard from distance...
+
+We admit it. It is partially our fault. Us, engineers.
```

Comment:
I would shorten the whole intro story into 1-2 sentence, as you have more then allowed characters.